### ISSUE TITLE
:config section is not running in spacemacs/init-google-translate 

### ISSUE THREAD

**Issue by lislon** [syl20bnr/spacemacs#4827]
I noticed a little issue in `spacemacs/init-google-translate` function. 

The code under `:config` keyword is not actually running when I hit any of commands listed in `:commands`.

 ```
(defun spacemacs/init-google-translate ()
  (use-package google-translate
    :commands (google-translate-query-translate
               google-translate-at-point
               google-translate-query-translate-reverse
               google-translate-at-point-reverse)
    :init
    (progn
      ;; some code ommited
      (spacemacs/set-leader-keys
        "xgQ" 'google-translate-query-translate-reverse
        "xgq" 'google-translate-query-translate
        "xgT" 'google-translate-at-point-reverse
        "xgt" 'google-translate-at-point))
    :config
    (progn
      (require 'google-translate-default-ui)
      (setq google-translate-enable-ido-completion t)
      (setq google-translate-show-phonetic t)
      (setq google-translate-default-source-language "en")
      (setq google-translate-default-target-language "fr"))))
```

The :config section is only executed when I explicitly say `(require 'google-translate)`. I suspect that this is due the functions listed in `:command` section are defined in *google-translate-default-ui.el*, not *google-translate.el*. 

*google-translate.el* consist of  just single require:
```
(require 'google-translate-default-ui)

(provide 'google-translate)
```

I am not sure how to fix it: Is it issue of `use-package` or `spacemacs/init-google-translate`?

**Comment by TheBB** [comment]
That's strange. The code seems fine if a bit odd (since those functions are autoloadable anyway).

**Comment by TheBB** [comment]
https://github.com/syl20bnr/spacemacs/pull/4829 fixes this for me. Can you check?

**Comment by lislon** [comment]
Working fine. Thank you.

**Comment by lislon** [comment]
But still the :config section is not working for some other customization.

**Comment by lislon** [comment]
Currenty google-translate keys are not working in develop. Merging #4912 will help

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
